pkg/features/kube_features_test.go

```diff
@@ -73,3 +73,16 @@ func TestAllRegisteredFeaturesExpected(t *testing.T) {
 		}
 	}
 }
+
+func TestEnsureAlphaGatesAreNotSwitchedOnByDefault(t *testing.T) {
```

Comment:
this part looks good to me... I'd maybe start with just this and a (hopefully temporary) exception for the one oddball gate to get the protection in place quick

Reply:
good point